client/src/components/Settings.tsx

```diff
@@ -89,6 +89,16 @@ const Settings: React.FC<Props> = ({
               }
               label={<Typography variant="subtitle1">Enable Asynchronous APIs</Typography>}
             />
+            <FormControlLabel
+              control={
+                <Checkbox
+                  name="postmanEnabled"
+                  checked={postmanEnabled}
+                  onChange={handleChange}
+                />
+              }
+              label={<Typography variant="subtitle1">Enable Postman Runtime</Typography>}
```

Comment:
Can we use more descriptive text on the purpose of Postman Runtime? 'Enable testing with Postman', for example.

What do you think?

Reply:
Done via new commit